.github/actions/free-disk-space/action.yml

```diff
@@ -42,8 +42,6 @@ runs:
       # moby/buildkit    latest      896276ced360   2 weeks ago    172MB
       run: |
         docker image rm \
-          node:14 \
```

Comment:
should we adjust the comment above to give the latest list of pre-installed images?

or is it the list here for `ubuntu-latest` ?
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#cached-docker-images

Reply:
updated, will merge once CI's happy